ios/RCTOneSignal/RCTOneSignal.m

```diff
@@ -74,28 +74,28 @@ - (void)sendEvent:(NSString *)eventName withBody:(NSDictionary *)body {
     [RCTOneSignalEventEmitter sendEventWithName:eventName withBody:body];
 }
 
-- (void)onOSPushSubscriptionChangedWithStateChanges:(OSPushSubscriptionStateChanges * _Nonnull)stateChanges {
-    [self sendEvent:OSEventString(SubscriptionChanged) withBody:[stateChanges.to jsonRepresentation]];
+- (void)onPushSubscriptionDidChangeWithState:(OSPushSubscriptionChangedState * _Nonnull)state {
```

Comment:
Send the ChangedState with previous and current

Reply:
Addressed in [Update PushSubscription interfaces, properties, and functions](https://github.com/OneSignal/react-native-onesignal/pull/1532/commits/3670e4f676443cbe5a6bf8fa1adaae8aa1edcf37)